lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfodProperties.td

```diff
@@ -4,4 +4,14 @@ let Definition = "symbollocatordebuginfod" in {
   def ServerURLs : Property<"server_urls", "Array">,
     ElementType<"String">,
     Desc<"An ordered list of Debuginfod server URLs to query for symbols. This defaults to the contents of the DEBUGINFOD_URLS environment variable.">;
+  def SymbolCacheLocation: Property<"cache_location", "String">,
+    DefaultStringValue<"">,
+    Desc<"The location where symbol files should be cached. This defaults to LLDB's system cache location.">;
+  def EnableAutoLookup : Property<"lookup_mode", "Enum">,
+    DefaultEnumValue<"eLookupModeAlways">,
+    EnumValues<"OptionEnumValues(g_debuginfod_symbol_lookup_mode)">,
+    Desc<"Control when DEBUGINFOD servers are queried for symbols">;
+  def Timeout : Property<"timeout", "UInt64">,
+    DefaultUnsignedValue<0>,
```

Comment:
`DefaultUnsignedValue<90>,`

Reply:
I left it at 0, and commented it, because the default of 90 seconds is handled inside the Debuginfod library.

Reply:
is it handled when you pass in a timeout of zero? If not, then set this to 90 for clarity?

Reply:
Zero will just always fail, so it's not a valid timeout. Is that a concern, usability-wise?